lib/main_containers.pm

```diff
@@ -31,36 +31,49 @@ sub is_container_test {
     get_var('CONTAINER_RUNTIME') ? return 1 : return 0;
 }
 
+sub is_container_image_test {
+    return get_required_var('FLAVOR') =~ /(Container-Image|NET|JeOS-for-kvm-and-xen)/;
+}
+
+sub is_res_host {
+    return get_var("HDD_1") =~ /(res82.qcow2|res79.qcow2)/;
```

Comment:
what is "res82" and "res79" ? where it is coming from ? I think we should have some commit id or url to confluence page giving some bigger context

Reply:
Just added a comment. I answered here for more info: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13349#discussion_r718411843